cmd/provider/main.go

```diff
@@ -20,12 +20,12 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
```

Comment:
nitpick: I don’t think this should be moved

Reply:
grouping non-repo ones together, also consistent with the rest of the repo as far as I could see.

Reply:
Typically we group org related ones together, but I guess that this one is _technically_ in a different org 👍